### PR TITLE
[ISSUE #4770] Beta publish: change the way of select betaIps, from input to select

### LINKED ISSUE #4770: Beta发布: 灰度发布不能选择客户端ip 需要手动填写，希望增加支持从已连接的客户端直接选择ip
Beta release: Grayscale release can not select client IP needs to be filled in manually, hope to add support to select IP directly from the connected client
Beta发布: 灰度发布不能选择客户端ip 需要手动填写，希望增加支持从已连接的客户端直接选择ip

### PR DESCRIPTION
**Pull request by brothelul**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

Fixes #4770 
change the way of select betaIps, from input to select.

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

File: console-ui/src/pages/ConfigurationManagement/ConfigEditor/NewConfigEditor.js [hunk 1]
```diff
@@ -493,11 +518,16 @@ class ConfigEditor extends React.Component {
                   </Checkbox>
                 )}
                 {isBeta && (
-                  <Input.TextArea
-                    aria-label="TextArea"
-                    placeholder="127.0.0.1,127.0.0.2"
-                    value={betaIps}
-                    onChange={betaIps => this.setState({ betaIps })}
+                  <Select
+                    size="medium"
```
Comment (haoyann) [inline console-ui/src/pages/ConfigurationManagement/ConfigEditor/NewConfigEditor.js:522]: I think use `tag` mode better, allow input as an option.
doc:https://fusion.design/pc/component/select?themeid=2

Comment (brothelul) [reply console-ui/src/pages/ConfigurationManagement/ConfigEditor/NewConfigEditor.js:522]: Yes, It's better to use tag mode if we allow users to input the betaIps which are not in the current Ip list.

**Comment by brothelul** [comment]
This PR conflicts with the develop branch, I will resolve the conflicts and re-submit later.

**Comment by brothelul** [comment]
the result after change:
origin
image
beta
image

**Review by haoyann — APPROVED** [review]

**Review by KomachiSion — APPROVED** [review]

**Pull request merged**